environment.yml

```diff
@@ -0,0 +1,17 @@
+name: kindle2Notion
```

Comment:
What is this file for? Seems like some `conda` stuff. If @paperboi is ok with that, please add some documentation about that, preferably to readme file.